### PR TITLE
Update FAQ with mentioning default_ssl support.

### PR DESCRIPTION
**Pull request by winksaville**
Here is an update for the FAQ for change [PR 2415](https://github.com/ponylang/ponyc/pull/2415)

### CODE REVIEW

File: content/faq/index.md [hunk 1]
```diff
@@ -137,12 +137,14 @@ By default, the Pony standard library uses OpenSSL 0.9 for various cryptography
...
 
-You can also encounter this problem if you try to build the Pony standard library tests via the ponyc Makefile. This can be addressed by adding `OPENSSL=-Dopenssl_1.1.` to the `make` command line:
+You can also change which openssl library will be used when compiling ponyc by passing `default_openssl=Name`. Where `Name` maybe `openssl_1.1.0` or `openssl_0.9.0`. For example, to set the default ssl library to `openssl_1.1.0` do the following:
 
 ```bash
-make OPENSSL=-Dopenssl_1.1.0 test
+make default_openssl=openssl_1.1.0
 ```
 
+The whatever value default_ssl has you can always override it by using the appropriate define, `ponyc -Dopenssl_0.9.0` or `ponyc -Dopenssl_1.1.0`.
```
Comment (SeanTAllen) [inline content/faq/index.md]: I'm going to update this to something that makes sense. i think i'll get it right, you should check.

Comment (SeanTAllen) [reply content/faq/index.md]: You didn't change the 

"By default, the Pony standard library uses OpenSSL 0.9 for various cryptography functions. This means if your OS has a non-default SSL library installed, you'll have to let ponyc know. For example Arch Linux has OpenSSL 1.1 installed. On such a system, when compiling your application, add `-Dopenssl_1.1.0` to the `ponyc` command:

```bash
ponyc -Dopenssl_1.1.0
```
"

isn't that no longer true?

**Comment by winksaville** [comment]
The -Dopenssl_xxx is still valid as a parameter to the compiler.

**Comment by winksaville** [comment]
Your change is great, txs.

**Pull request merged**